packages/main/src/components/VariantManagement/ManageViewsDialog.tsx

```diff
@@ -159,7 +167,7 @@ export const ManageViewsDialog = (props: ManageViewsDialogPropTypes) => {
         deletedRows: deletedTableRows.current
       });
     } else {
-      Object.values(invalidVariants)[0].focus();
+      Object.values(invalidVariants).forEach((invalidVariant) => invalidVariant.focus());
```

Comment:
Why has this been changed? This will set the focus on every invalid variant, which is not the desired behavior, since only one element should be focused at a time.

Reply:
The reason was because it seemed easier to see all invalid variants at the same time. Would you prefer the one at a time behaviour?

Reply:
Yup, because there can only ever be one activeElement and so I think we should start with the first invalid variant.